### ISSUE TITLE
Fatal: submitJoin: our node was reported to not be reachable by 2 or more nodes 5

### ISSUE THREAD

**Issue by tm324462** [Shardeum/shardeum-bug-reporting#468]
Node exited with the following message:
Fatal: submitJoin: our node was reported to not be reachable by 2 or more nodes 5

**Comment by yeidy2** [comment]
Fatal: submitJoin: our node was reported to not be reachable by 2 or more nodes 5

**Comment by yeidy2** [comment]
> Node exited with the following message: Fatal: submitJoin: our node was reported to not be reachable by 2 or more nodes 5

higher docker compose version

**Comment by tm324462** [comment]
shm v1.9.0
docker-compose version 1.29.2, build 5becea4c
docker-py version: 5.0.0
CPython version: 3.7.10
OpenSSL version: OpenSSL 1.1.0l  10 Sep 2019

**Comment by jorisc90** [comment]
Same issue here. I definitely did forward ports TCP 9001 / 10001 to the device running Shardeum.
```
root@shardeum:~# docker-compose version
docker-compose version 1.29.2, build unknown
docker-py version: 5.0.3
CPython version: 3.10.12
OpenSSL version: OpenSSL 3.0.2 15 Mar 2022

root@shardeum:~# docker version
Client:
 Version:           24.0.5
 API version:       1.43
 Go version:        go1.20.3
 Git commit:        24.0.5-0ubuntu1~22.04.1
 Built:             Mon Aug 21 19:50:14 2023
 OS/Arch:           linux/amd64
 Context:           default

Server:
 Engine:
  Version:          24.0.5
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.3
  Git commit:       24.0.5-0ubuntu1~22.04.1
  Built:            Mon Aug 21 19:50:14 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.7.2
  GitCommit:
 runc:
  Version:          1.1.7-0ubuntu1~22.04.1
  GitCommit:
 docker-init:
  Version:          0.19.0
  GitCommit:

root@shardeum:~# uname -a
Linux shardeum 6.5.11-7-pve #1 SMP PREEMPT_DYNAMIC PMX 6.5.11-7 (2023-12-05T09:44Z) x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by tm324462** [comment]
nobody？

**Comment by swarna1101** [comment]
update to the latest node its solved in latest version 1.9.5.
